src/sacn_parse_pack_error.rs

```diff
@@ -9,162 +9,55 @@
 
 /// The errors used within the SacnLibrary specifically those related to parsing and packeting packets received/sent on the network.
 ///
+
 pub mod sacn_parse_pack_error {
-    error_chain! {
-        errors {
-            /// When parsing packet invalid data encountered.
-            ///
-            /// # Arguments
-            /// msg: A message providing further details (if any) as to what data was invalid.
-            ///
-            ParseInvalidData(msg: String) {
-                description("Data provided to parse into a packet is invalid"),
-                display("Error when parsing data into packet, msg: {}", msg)
-            }
-
-            /// Attempted to parse a priority value that is outwith the allowed range of [0, E131_MAX_PRIORITY].
-            /// As per ANSI E1.31-2018 Section 6.2.3
-            ///
-            /// # Arguments
-            /// msg: A message providing further details (if any) as to why the priority valid was invalid.
-            ///
-            ParseInvalidPriority(msg: String) {
-                description("Attempted to parse a priority value that is outwith the allowed range of [0, 200]"),
-                display("Attempted to parse a priority value that is outwith the allowed range of [0, 200], msg: {}", msg)
-            }
-
-            /// Attempted to parse a page value that is invalid - e.g. the page value is higher than the last_page value.
-            ///
-            /// # Arguments
-            /// msg: A message providing further details (if any) as to why the page was invalid.
-            ///
-            ParseInvalidPage(msg: String) {
-                description("Attempted to parse a page value that is invalid"),
-                display("Error when parsing page value, msg: {}", msg)
-            }
-
-            /// Attempted to parse a sync address value that is outwith the allowed range of [0, E131_MAX_MULTICAST_UNIVERSE].
-            /// As per ANSI E1.31-2018 Section 9.1.1.
-            ///
-            /// # Arguments
-            /// msg: A message providing further details (if any) as to why the synchronisation address was invalid.
-            ///
-            ParseInvalidSyncAddr(msg: String) {
-                description("Attempted to parse a sync_addr value that is outwith the allowed range of [0, 63999]"),
-                display("Attempted to parse a sync_addr value that is outwith the allowed range of [0, 63999], msg: {}", msg)
-            }
-
-            /// Attempted to parse a universe value that is outwith the allowed range of [1, E131_MAX_MULTICAST_UNIVERSE].
-            /// As per ANSI E1.31-2018 Section 9.1.1.
-            ///
-            /// # Arguments
-            /// msg: A message providing further details (if any) as to why the universe field was invalid.
```

Comment:
we don't want to lose this documentation particularly the bits that relate back to the spec